dev-tools/packaging/templates/docker/Dockerfile.tmpl

```diff
@@ -57,7 +57,6 @@ RUN for iter in {1..10}; do \
 {{- end }}
 
 {{- if contains .from "ubuntu" }}
-RUN touch /var/mail/ubuntu && chown ubuntu /var/mail/ubuntu && userdel -r ubuntu
```

Comment:
Was also introduced by the upgrade https://github.com/elastic/beats/pull/40942
The older Ubuntu image does not build with this.